src/api/public/apidocs-new/paths/person_login.yaml

```
@@ -0,0 +1,37 @@
+get:
+  summary: Get details about a person
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: path
+      name: login
+      schema:
+        type: string
+      required: true
+      description: The person login name
+  responses:
+    '200':
+      description: >
+        OK. The request has succeeded.
+
+        XML Schema used for body validation: [person.xsd](../schema/person.xsd)
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/person.yaml'
+          example:
+            login: Admin
+            email: root@localhost
+            realname: OBS Instance Superuser
+            state: confirmed
+            globalrole: Admin
+            ignore_auth_services: false
+            watchlist:
+              - project:
+                  name: "home:Admin"
+    '401':
+      $ref: '../components/responses/unauthorized.yaml'
+    '404':
+      $ref: '../components/responses/notfound.yaml'
```

```suggestion
      $ref: '../components/responses/not_found.yaml'
```
Using a not found response file works only in this case, because of the `summary` string in the example. Unfortunately, the component can not be reused, because the message in the example will be not always the same.

I'm for not using a component file here.

In case of using a file, I would also rename it adding an underscore.

---

I would use a component and assume the message is just an example.

---

I understand both points of view. The example's `summary` is not appropriate for all the `not_found` errors we can get. However, the example can be overwritten when defining the errors... but we can't ensure that everyone overwrites the example.

What about using a generic `summary` like: 
```xml
<summary> <summary> </summary>
```
?

Then, if someone forgets to overwrite the example, we won't have an inappropriate `summary`.

As plan B, we can also remove the `summary` from the file. In the worst case, we'll have a missing `summary`, not an inappropriate one.

---

Sorry, I assumed the example can be overwritten, but it can not. I was mixing up `schemas` with `responses`.

We have decided offline to use some specific not_found files inside a proper namespace.